### ISSUE TITLE
alarm limits slider editor

### ISSUE THREAD

**Issue by cicardia** [andrefbsantos/boilr#81]
Joel: when changing the alarm bounds why not also have a slider to increase or decrease the value?

We talked about this earlier. Most users won't know how to properly use the slider and we wouldn't have space for the text and slider right? or is there a mix?

**Comment by dllud** [comment]
[Sliders aka Seek Bars](https://developer.android.com/design/building-blocks/seek-bars.html) are utterly difficult to use when you need to set a precise value. Such problem grows with the range size, the bigger the range the larger the amount of values condensed in the seek bar.

Now the real issue here is that Boilr's price ranges are infinite, both in something we may call precision (e.g. they can be this: 1001.01 or this 1001.00000000001) and in something we may call value (they can be this: 3030 or this: 99867532632793294).

To make a price seek bar somewhat usable we would have to guess its range and precision. This means we would have to grab the last value, say 0.001479, put on our wizard hat and go for a guess:
> Umm... lets see, this last value has 6 decimal digits, thus lets set a 0.000001 precision. Now... we are dealing with an upper limit here, thus our range will start on 0.001479. Finally, more than 100 points in the bar makes it too difficult to use, thus lets end our range at 0.001579.

Now I would like to know the % of users creating such alarm that would have their pretended value inside that range AND prefer to use a seek bar instead of a text input.

Yes, if we ever use something like this we must custom build a number picker with a seek bar and a text input right by its side (I could find no library providing this on [Android Arsenal - Range Bars](https://android-arsenal.com/tag/83)). But wouldn't that be too much work for nothing? Who would use the seek bar more than once? Writing a number with the numeric keypad is damn easy.

Leaving this open to heard what others have to say.

**Comment by cicardia** [comment]
I agree, just shared Joel's opinion. But imho this would be a mismanagement of coding hours. It's less than b&whistles
-- 
<a href="http://cargocollective.com/algazarra" target="_blank">Algazarra</a>
